public/app/features/explore/state/query.ts

```diff
@@ -633,17 +635,20 @@ export const runQueries = createAsyncThunk<void, RunQueriesOptions>(
 
       newQuerySubscription = newQuerySource.subscribe({
         next(data) {
+          const exploreState = getState().explore.panes[exploreId];
           if (data.logsResult !== null && data.state === LoadingState.Done) {
             reportInteraction('grafana_explore_logs_result_displayed', {
               datasourceType: datasourceInstance.type,
+              visualisationType:
+                exploreState?.panelsState?.logs?.visualisationType ?? store.get(visualisationTypeKey) ?? 'N/A',
```

Comment:
While we are at it, do we want to add information about the number of logs displayed? Something like 
```suggestion
                exploreState?.panelsState?.logs?.visualisationType ?? store.get(visualisationTypeKey) ?? 'N/A',
              length: data.logsResult.rows.length,
```